ssms/basic_simulators/simulator.py

```diff
@@ -513,6 +513,7 @@ def simulator(
         "ornstein_uhlenbeck",
         "ornstein_angle",
         "gamma_drift",
+        "shrink_spot",
```

Comment:
missing the part where you assign a particular `sigma` (`s` parameter).
can you confirm that sigma is set to the correct value for this simulator, and that the parameter bounds are the correct ones for that particular sigma?

Reply:
Yes, the sigma value for shrink_spot is 1.0. The bounds that I have specified in config are the ones specified in the paper, where p_target/outer/inner are scaled by 10. "a" was already between 0.3 and 3.0, which covered the bounds specified in that paper already.